tests/Functional/Action/Lti/GetUserAssignmentLtiLinkActionTest.php

```diff
@@ -203,7 +207,7 @@ public function testItReturnsLtiLinkAndUpdatedAssignmentAndAttemptsCountWithUser
                     'oauth_timestamp' => (string)Carbon::getTestNow()->getTimestamp(),
                     'oauth_version' => OAuthContext::VERSION_1_0,
                     'lti_message_type' => LtiRequest::LTI_MESSAGE_TYPE,
-                    'lti_version' => LtiRequest::LTI_VERSION,
+                    'lti_version' => LtiRequest::LTI_VERSION_1P0,
```

Comment:
I'd suggest using a data provider to test both cases 1.1 and 1.3

Reply:
We definitely could, but in this case they are built in a completely different way, so I don't see a easy way to use providers